styles.css

```diff
@@ -0,0 +1,141 @@
+
+body {
+  background: url(images/background.jpg);
+  background-size: cover;
+
+}
+
+#exphr {
+height: 3px;
+color: #123455;
+background-color: #800000;
+border: none;
+}
+
+#expheader {
+  padding-top: 50px;
+}
+
+#aboutmebg {
+  background: url(none);
+  background-color: #ffffff;
+  text-align: center;
+}
+
+#h1bio {
+  color: black;
+  font-size: 50px;
+}
+
+#expBG {
+  background: url(images/kurt_3.jpg);
+  background-size: cover;
+}
+
+h1 {
+  font-size: 50px;
+  color: #800000;
+}
+
+h4 {
+  font-size: 50px;
+  color: #800000;
+  text-align: center;
+  font-family: 'Barrio', cursive;
+}
+
+h2 {
+  font-family: 'Indie Flower', cursive;
+  margin-left: 10px;
+  
+
+}  
+
+#h1home {
+  padding-bottom: 250px;
+}
+
+
+header {
+  text-align: center;
+  padding-bottom: 20px;
+}
+
+#coverphoto {
+  border-radius: 40%;
+  display: block;
+  margin: auto;
+  width: 27%;
+  padding-bottom: 20px;
+}
+
+h3 {
+  color: #800000;
+  font-size: 30px;
+  font-family: 'Barrio', cursive;
+}
+
+a {
+  text-decoration: none;
+}
+
+#topnavs {
+
+}
+
+.button {
+    background-color: black;
+    color: #800000;
+    border: 2px solid #555555;
+    border-radius: 12px;
+    font-size: 30px;
+    padding: 10px 10px 10px 10px;
+
+
+}
+
+.button:hover {
+    background-color: #555555;
+    color: white;
+    font-size: 30px;
+    padding: 10px 10px 10px 10px;
+}
+
+.button1 {
+    background-color: white;
+    color: black;
+    border: 2px solid #555555;
+    border-radius: 12px;
+    font-size: 30px;
+    padding: 10px 10px 10px 10px;
+}
+
+
+.button1:hover {
+    background-color: black;
+    color: white;
+    font-size: 30px;
+    padding: 10px 10px 10px 10px;
+}
+
+#link {
+    background-color: white;
+    color: black;
+    border: 3px solid #555555;
+    font-size: 40px;
+}
+
+#link:hover {
```

Comment:
Great attention to detail utilizing pseudo-selectors to take advantage of different user interactions. 👍  Should be a class though so the style can be applied across all similar elements through out your site.  This method makes you have to style each individually, defeating the main benefit of CSS and the best practice of keeping your code DRY (Don't Repeat Yourself).